site/docs/spec.md

```diff
@@ -581,10 +596,11 @@ Table metadata consists of the following fields:
 | _optional_ | _optional_ | **`metadata-log`**| A list (optional) of timestamp and metadata file location pairs that encodes changes to the previous metadata files for the table. Each time a new metadata file is created, a new entry of the previous metadata file location should be added to the list. Tables can be configured to remove oldest metadata log entries and keep a fixed-size log of the most recent entries after a commit. |
 | _optional_ | _required_ | **`sort-orders`**| A list of sort orders, stored as full sort order objects. |
 | _optional_ | _required_ | **`default-sort-order-id`**| Default sort order id of the table. Note that this could be used by writers, but is not used when reading because reads use the specs stored in manifest files. |
+|            | _optional_ | **`refs`** | A list of snapshot references, stored as full snapshot reference objects. |
+|            | _optional_ | **`current-branch`** | The name of the current branch. If not specified, it defaults to the `main` branch that starts with the table creation commit. | 
```

Comment:
Should we have a `current_branch` in metadata? Will it make more sense to position this as a client-session specific conf?

Reply:
I think the metadata need to know this information, because it will use the current branch to know what is the default branch to add the next commit.